### PR TITLE
editTask

### PR DESCRIPTION
**Pull request by tmndung**

### CODE REVIEW

**Comment by tmndung** [comment]
ready

**Comment by tmndung** [comment]
ready

**Comment by tmndung** [comment]
ready

**Comment by tmndung** [comment]
ready

File: resources/views/tasks/update.blade.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+                <div class="panel-body">
+
+                    @include('common.errors')
+
+                      {!! Form::open(['url' => ('tasks/' .$task->id), 'method' => 'POST', 'class' => 'form-horizontal', 'method' => 'PATCH']) !!}
+                        <div class="form-group">
+                             {!! Form::label('task-name', trans('message.edit_task'), ['class' => 'col-sm-3 control-label']) !!}
+
+
```
Comment (tranducquoc) [inline resources/views/tasks/update.blade.php]: remove 2 line này

File: resources/views/tasks/update.blade.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                        <div class="form-group">
+                            <div class="col-sm-offset-3 col-sm-6">
+                                {{ Form::submit(trans('message.update_task'), ['class' => 'btn btn-default']) }}
+                            </div>
+                        </div>
+                    {!! Form::close() !!}
+                </div>
+            </div>
+
+
```
Comment (tranducquoc) [inline resources/views/tasks/update.blade.php]: remove 2 line này

**Comment by tranducquoc** [comment]
commented

**Comment by tmndung** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**